cli/ops/process.rs

```diff
@@ -86,28 +86,37 @@ fn op_run(
   }
 
   // TODO: make this work with other resources, eg. sockets
-  let stdin_rid = run_args.stdin_rid;
-  if stdin_rid > 0 {
-    let file = clone_file(stdin_rid, &mut resource_table)?;
-    c.stdin(file);
+  if run_args.stdin != "" {
+    if let Some(stdin) = subprocess_stdio_map(run_args.stdin.as_ref()) {
+      c.stdin(stdin);
+    } else {
+      return Err(OpError::other("Invalid resource for stdin".to_string()));
```

Comment:
Maybe lift this error into the utility function subprocess_stdio_map, so it doesn't have to be repeated